packages/core/src/components/content-header/content-header.scss

```
@@ -18,12 +18,11 @@
     flex-direction: column;
     flex: 1 1 0;
     min-width: 0;
-    white-space: nowrap;
     margin-right: 0.5rem;
 
     .titleOverflow {
-      overflow: hidden;
-      text-overflow: ellipsis;
+      overflow-wrap: break-word;
+      word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)